megamek/src/megamek/client/ui/swing/skinEditor/SkinSpecPanel.java

```diff
@@ -648,9 +641,6 @@ public void setupSkinEditPanel(SkinSpecification skinSpec) {
 
     @Override
     public void valueChanged(ListSelectionEvent e) {
-        if (e.getValueIsAdjusting()) {
```

Comment:
?

Reply:
Both cases just immediately return, so it's just bad handling